api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/BaseServiceRequestTranslatorImpl.java

```diff
@@ -15,6 +15,7 @@
 
```

Comment:
Not a point for this PR, but maybe we should talk through this class. The more I look at it, the more I feel like its pretty tightly coupling `ServiceRequest` to `Task` which makes sense in the lab workflow, but I don't think is a good, generalizable thing.